core/tests/puppeteer-acceptance-tests/user-utilities/question-admin-utils.js

```diff
@@ -0,0 +1,311 @@
+  /**
+   * Function for navigating to the contributor dashboard admin page.
+   */
+  async navigateToContributorDashboardAdminPage() {
+    await this.goto(contributorDashboardAdminUrl);
+  }
+
+  /**
+   * Function for adding a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async addReviewQuestionRights(username) {
+    await this.type(addContributorUsernameInput, username);
+    await this.select(
+      addContributonRightsCategorySelect, reviewQuestionRightValue);
+    await this.clickOn(addContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for adding a right of submitting questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async addSubmitQuestionRights(username) {
+    await this.type(addContributorUsernameInput, username);
+    await this.select(
+      addContributonRightsCategorySelect, submitQuestionRightValue);
+    await this.clickOn(addContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for removng a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async removeReviewQuestionRights(username) {
+    await this.type(removeContributorUsernameInput, username);
+    await this.select(
+      removeContributonRightsCategorySelect, reviewQuestionRightValue);
+    await this.clickOn(removeContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function for removing a right of reviewing questions to a user.
+   * @param {string} username - the username of the user.
+   */
+  async removeSubmitQuestionRights(username) {
+    await this.type(removeContributorUsernameInput, username);
+    await this.select(
+      removeContributonRightsCategorySelect, submitQuestionRightValue);
+    await this.clickOn(removeContributionRightsSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to display the list of question reviewers
+   */
+  async getDisplayedListOfQuestionReviewers() {
+    await this.select(viewContributorFilterMethodSelect, roleMethodValue);
+    await this.select(viewContributorCategorySelect, reviewQuestionRightValue);
+    await this.clickOn(viewContributorSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to display the list of question reviewers
+   */
+  async getDisplayedListOfQuestionSubmitters() {
+    await this.select(viewContributorFilterMethodSelect, roleMethodValue);
+    await this.select(viewContributorCategorySelect, submitQuestionRightValue);
+    await this.clickOn(viewContributorSubmitButton);
+
+    await this.page.waitForNetworkIdle();
+  }
+
+  /**
+   * Function to display the contribution rights status for the user.
+   * @param {string} username - the username of the user.
+   */
+  async contributionStatusForUser(username) {
```

Comment:
Please start all function names with verbs. 

Perhaps this one could also extract and return the user's contribution status.

Reply:
Done. This function is also extracted and now it returns an object/element that contains the user's contribution status.